main.go

```diff
@@ -39,63 +41,81 @@ import (
 	"github.com/sonatype-nexus-community/nancy/parse"
 )
 
+var appLog = logger.Logger
+
 func main() {
+	appLog.Debug("Starting Nancy")
+
 	if len(os.Args) > 1 && os.Args[1] == "iq" {
+		appLog.Debug("Nancy parsing config for IQ")
 		config, err := configuration.ParseIQ(os.Args[2:])
 		if err != nil {
 			flag.Usage()
 			os.Exit(1)
 		}
 		processIQConfig(config)
+		appLog.Debug("Nancy finished parsing config for IQ")
```

Comment:
Do we want even more printed here as far as the actually config that was parsed goes??

Reply:
I deliberately avoided printing config as it could have SENSITIVE info.

Reply:
Is it worthwhile to redact the sensitive info?? Passwords, urls, etc as part of logging?? A logrus hook like this could help. 

https://github.com/whuang8/redactrus

Reply:
Redactrus works only on named fields, not on an object that you pass to logrus, sadly. It's also 2 years old and looks to be not maintained. Might be fun to cook something like that up!